src/components/LHNOptionsList/LHNOptionsList.tsx

```diff
@@ -72,7 +68,7 @@ function LHNOptionsList({
                     reportActions={itemReportActions}
                     parentReportAction={itemParentReportAction}
                     policy={itemPolicy}
-                    personalDetails={participantsPersonalDetails}
+                    personalDetails={personalDetails ?? undefined}
```

Comment:
```suggestion
                    personalDetails={personalDetails ?? {}}
```
While `personalDetails` should never be `undefined` or `null`, I think it is unsafe to pass explicitly `undefined` like this. If we pass `undefined` explicitly this way, default value for `personalDetails` in `OptionRowLHNData` will not be used. I think it is better to pass `{}` here instead of `undefined`.

Reply:
In `OptionRowLHNData` it is used only for `getOptionData` and it should not cause an issue even when it is `undefined`. But I think it is better if we never pass `undefined` here especially when we are giving a default value of {} for `personalDetails` in `OptionRowLHNData`. WDYT?

Reply:
I agree.

Reply:
@c3024 

Updated and merged with the latest main